### ISSUE TITLE
Radio buttons are hard or impossible to access via keyboard (browser-dependent)

### ISSUE THREAD

**Issue by toolness** [truetandem/e-QIP-prototype#1339]
These kinds of radio buttons are, for some reason, difficult to focus with the keyboard via <kbd>Tab</kbd>, depending on one's browser:

> ![2017-05-30_18-14-12](https://cloud.githubusercontent.com/assets/124687/26607316/e9a0e3b4-4563-11e7-8812-e7055ae00524.png)

* On Firefox, the buttons are keyboard-focusable when neither is selected. However, once one of the buttons is chosen, both become inaccessible via keyboard.

* On Chrome, behavior is *extremely* weird: in the above example, the "Yes" button is only focusable when moving *forwards* through the tab ordering, and the "No" button is only focusable when moving *backwards* through the tab ordering.

* On Safari, the buttons are not keyboard-focusable at all.

As with #1335, I wasn't able to focus on these buttons at all via the NVDA or VoiceOver screenreaders.

I guess this is probably the most urgent of all of the accessibility-related issues I've filed recently, since it means that keyboard and screenreader-based users may not be able to answer questions at all!

**Comment by colinpmacarthur** [comment]
@bmallred What's the intended functionality here? I am still seeing the same behavior in Chrome.

**Comment by bmallred** [comment]
For an example going to the ***Other Names*** section

 1. Click on **yes**
 2. Press **tab** on the keyboard and "no" should be selected

Just tested on Chrome and Firefox and working as expected. Was there other behavior I should be testing?

cc: @colinpmacarthur @Jay-Neighbours-I72 @myukna72

**Comment by colinpmacarthur** [comment]
Ah, got it. This makes sense to me now. Thanks!

**Comment by Jay-Neighbours-I72** [comment]
Looks good to me too.

**Comment by billy22g** [comment]
LGTM
